src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -366,6 +375,7 @@ protected override void Free()
             _updatedSymbolMapOpt = updatedSymbolMapOpt;
             _returnTypesOpt = returnTypesOpt;
             _snapshotBuilderOpt = snapshotBuilderOpt;
+            _defaultValues = defaultValues;
```

Comment:
>defaultValues [](start = 29, length = 13)

from our discussion, is it possible to initialize the empty dictionary here? When do we need a dictionary passed in across analyses?

Reply:
It seems that we don't, I somehow got the impression that we did when I was troubleshooting the bootstrap build

---
In reply to: [329245259](https://github.com/dotnet/roslyn/pull/38788#discussion_r329245259) [](ancestors = 329245259)